libminifi/include/FlowController.h

```diff
@@ -339,6 +339,12 @@ class FlowController : public core::controller::ControllerServiceProvider, publi
    */
   virtual void initializePaths(const std::string &adjustedFilename);
 
+  template <typename T, typename = typename std::enable_if<std::is_base_of<SchedulingAgent, T>::value>::type>
+  void conditionalReloadScheduler(std::shared_ptr<T>& scheduler, const bool condition) {
```

Comment:
would make this template private 
(as a matter of fact, I would make this whole protected block private, and use the discussed TestAccessor method, but that can wait, as I believe that would be out of scope for this PR)

Reply:
Changed this to private.